From 5a39b9f4f99cb4076869a30aa90d8a26ccbb601c Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Bernhard=20K=C3=B6lbl?= <bkoelbl@codeweavers.com>
Date: Wed, 29 Apr 2026 11:15:56 +0200
Subject: [PATCH 03/47] windows.media.speech: Fix a trace typo.

(cherry picked from commit 1d744d4ca948c7ea403a786f21c5fe4ef6de08fe)

CW-Bug-Id: #27011
---
 dlls/windows.media.speech/synthesizer.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/dlls/windows.media.speech/synthesizer.c b/dlls/windows.media.speech/synthesizer.c
index d9fec2c651b..45a0076363b 100644
--- a/dlls/windows.media.speech/synthesizer.c
+++ b/dlls/windows.media.speech/synthesizer.c
@@ -115,7 +115,7 @@ static HRESULT WINAPI voice_information_get_DisplayName( IVoiceInformation *ifac
 {
     struct voice_information *impl = impl_from_IVoiceInformation(iface);
 
-    TRACE("iface %p, value %p!n", iface, value);
+    TRACE("iface %p, value %p\n", iface, value);
     return WindowsDuplicateString(impl->display_name, value);
 }
 
-- 
2.54.0

